Assets/Scripts/Content/StringI18n.cs

```diff
@@ -0,0 +1,236 @@
+        /// Set the languages list from a comma separed string with them.
+        /// </summary>
+        /// <param name="stringLanguages">comma separed string</param>
+        public static void setLanguages(String stringLanguages)
+        {
+            languages = stringLanguages.Split(COMMA);
+            // Default language is english = 1. 
+            // Language 0 is the key
+            currentLanguage = 1;
+        }
+
+        /// <summary>
+        /// Sets current language using the string of the language.
+        /// If there is no language with this name, the default language
+        /// remains the same.
+        /// </summary>
+        /// <param name="languageName">Name of the language</param>
+        public static void setCurrentLanguage(String languageName)
+        {
+            for (int pos = 1; pos < languages.Length; pos++)
+            {
+                if (languages[pos] == languageName)
+                {
+                    currentLanguage = pos;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Sets curreng language using its position. 0 doesn't count.
+        /// </summary>
+        /// <param name="languagePosition"></param>
+        public static void setCurrentLanguage(int languagePosition)
+        {
+            if (languagePosition != 0)
+            {
+                currentLanguage = languagePosition;
+            }
+        }
+
+        /// <summary>
+        /// Instance info of the current translations
+        /// </summary>
+        private String[] translations;
+
+        /// <summary>
+        /// Creates an empty instance of a Multilanguage String
+        /// </summary>
+        public StringI18n()
+        {
+            translations = new String[languages.Length];
+        }
+
+        private const char QUOTES = '\"';
+        private const char COMMA = ',';
+
+        /// <summary>
+        /// Constructor with the complete localisation elements
+        /// </summary>
+        /// <param name="completeLocalisationString"></param>
+        public StringI18n(String completeLocalisationString)
+        {
+            if (completeLocalisationString.Contains(QUOTES))
+            {
+                // with quotes, commas inside quotes isn't considered separator
+                List<String> partialTranslation = new List<String>(completeLocalisationString.Split(COMMA));
+                List<String> finalTranslation = new List<string>();
+                String currentTranslation = "";
+                bool opened = false;
+                foreach (String suposedTranslation in partialTranslation)
+                {
+                    currentTranslation += suposedTranslation;
+
+                    bool initialQuote = suposedTranslation.Length != 0 && suposedTranslation[0] == QUOTES;
+                    bool finalQuote = suposedTranslation.Length > 1 &&
+                        suposedTranslation[suposedTranslation.Length - 1] == QUOTES;
```

Comment:
This method may not work because \" is escaped by using "" in the csv.  This means that:

"My text has a ""quote"", and other things"

Won't work right.